department-of-veterans-affairs/abd-vro
# MCP-2061 Evidence Summary PDF Updates

## What was the problem?
More updates to the PDF's design


Associated tickets or Slack threads:

- [MCP-2061](https://amida.atlassian.net/browse/MCP-2061)

## How does this fix it?[^1]


## How to test this PR
- Spot check some differences between the PDF in the ticket vs the generated one or verify via code

![image](https://user-images.githubusercontent.com/13013464/204562661-a49f0758-5642-474d-81f6-4dfc8836e7c0.png)
![image](https://user-images.githubusercontent.com/13013464/204562763-b3519967-e40f-4457-8119-108b1d0ae195.png)
![image](https://user-images.githubusercontent.com/13013464/204562928-978a24cf-f075-4445-ac26-ecdeb0053947.png)
